src/otx/core/data/entity/base.py

```diff
@@ -206,6 +207,143 @@ def __repr__(self) -> str:
         )
 
 
+class VideoInfo(tv_tensors.TVTensor):
+    """Meta info for image.
+
+    Attributes:
+        img_id: Image id
+        img_shape: Image shape (heigth, width) after preprocessing
+        ori_shape: Image shape (heigth, width) right after loading it
+        padding: Number of pixels to pad all borders (left, top, right, bottom)
+        scale_factor: Scale factor (height, width) if the image is resized during preprocessing.
+            Default value is `(1.0, 1.0)` when there is no resizing. However, if the image is cropped,
+            it will lose the scaling information and be `None`.
+        normalized: If true, this image is normalized with `norm_mean` and `norm_std`
+        norm_mean: Mean vector used to normalize this image
+        norm_std: Standard deviation vector used to normalize this image
+        image_color_channel: Color channel type of this image, RGB or BGR.
+        ignored_labels: Label that should be ignored in this image. Default to None.
```

Comment:
Could you align these docstring to current attirbutes?

Reply:
Updated at b92e52520aed9a19150f81e80ea272b1e4135155